internal/cmd/config.go

```diff
@@ -560,6 +560,8 @@ func Configure(v *viper.Viper, p *pflag.FlagSet) {
 	v.SetDefault("cadence::host", "")
 	v.SetDefault("cadence::port", 7933)
 	v.SetDefault("cadence::domain", "pipeline")
+	v.SetDefault("cadence::createNonexistentDomain", true)
+	v.SetDefault("cadence::workflowExecutionRetentionPeriodInDays", 3)
```

Comment:
**Required**: these 2 are already set in the `cmd/pipeline/config.go` and `cmd/worker/config.go`, for now we **MUST** leave them there with those values (createNonExistentDomain is set to false and is overridden in the installer).